cfgov/preprocessed/js/molecules/Expandable.js

```diff
@@ -0,0 +1,251 @@
+'use strict';
+
+// Required polyfills for <IE9.
+require( '../modules/polyfill/query-selector' );
+require( '../modules/polyfill/event-listener' );
+require( '../modules/polyfill/class-list' );
+
+// Required modules.
+var EventObserver = require( '../modules/util/EventObserver' );
+var $ = require( 'jquery' );
+
+/**
+ * Expandable
+ * @class
+ *
+ * @classdesc Initializes a new Expandable molecule.
+ *
+ * @param {Object} context
+ *   The DOM element within which to search for the molecule.
+ * @returns {Object} An Expandable instance.
+ */
+function Expandable( context ) { // eslint-disable-line max-statements, inline-comments, max-len
+
+  // TODO: Remove this when difference between atomic Expandable
+  //       and cf-expandables is resolved.
+  // ==========================================================================
+  // Unsets global expandables logic.
+  if ( $.fn.expandable ) {
+    var html = context.innerHTML;
+    context.innerHTML = html;
+
+    var $this = $( context );
+    var $cueOpen = $this.find( '.expandable_cue-open' ).not( $this.find( '.expandable .expandable_cue-open' ) );
```

Comment:
See https://github.com/cfpb/cfgov-refresh/pull/1134#discussion_r43973046, I'll get rid of this override.